### PR TITLE
4.0.0 NEWS

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+also require updates to module code. Refer to
+[4.0.0 Deprecations and breaking changes](https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes)
+
+Issues pertaining to this release (some of which are related):
+
+ * [EDGCOMMON-5](https://issues.folio.org/browse/EDGCOMMON-5) Merge EdgeVerticle2 -> EdgeVerticle
+ * [EDGCOMMON-19](https://issues.folio.org/browse/EDGCOMMON-19) Update dependencies
+ * [EDGCOMMON-30](https://issues.folio.org/browse/EDGCOMMON-30) Update to Vert.x 4
+ * [EDGCOMMON-31](https://issues.folio.org/browse/EDGCOMMON-31) Allow non-HTTP server
+ * [EDGCOMMON-32](https://issues.folio.org/browse/EDGCOMMON-32) EdgeVerticleHttp and update documentaion
```
Comment (hjiebsco) [inline NEWS.md:19]: Another typo `documentaion`. Is it a proofreading test? :)

**Review by hjiebsco — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (adamdickmeiss) [reply NEWS.md:19]: I guess I just need an editor that highlights simple typos.. My vi don't.

Comment (adamdickmeiss) [reply NEWS.md:19]: I'll fix the typo afterwards. Hope that's OK.

Comment (hjiebsco) [reply NEWS.md:19]: Of course. I was just kidding about the proofreading.

**Pull request merged**